releasenotes/notes/add-how-to-listen-on-ports-below-1024-9e65de4420d44d7d.yaml

```diff
@@ -0,0 +1,4 @@
+other:
+  - |
+    Add the comment to explain how to listen on the port under
+    1024 in pkg/config/config_template.yaml.
```

Comment:
```suggestion
    Add a comment to explain how to listen on ports under
    1024 in pkg/config/config_template.yaml.
```

Reply:
Is it necessary to add a changelog entry for an added comment?  Do customers benefit?

Reply:
You can set the `no-changelog` label to avoid that.

Reply:
Thanks, removed a changelog https://github.com/DataDog/datadog-agent/pull/14025/commits/e415462c3381e55be7a2b951cd7becccd2346a09